server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -68,6 +78,48 @@
 import io.restassured.specification.RequestSpecification;
 
 public abstract class DeletedMessagesVaultTest {
+
+    public static class FileSystemProbeImpl implements GuiceProbe {
```

Comment:
IMO wrong way to do it.

We should rather override the filesystem component with our own.

I propose that the `createJmapServer` takes a FileSystem argument